keyboards/yushakobo/primer61/info.json

```diff
@@ -0,0 +1,92 @@
+{
+    "manufacturer": "yushakobo",
+    "keyboard_name": "primer61",
+    "maintainer": "yushakobo",
+    "bootloader": "caterina",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x3265",
```

Comment:
Vendor ID appears to check out. 
https://www.the-sz.com/products/usbid/index.php?v=0x3265&p=&n=